packages/core/src/domain/configuration.spec.ts

```diff
@@ -160,6 +162,21 @@ describe('configuration', () => {
       expect(configuration.isIntakeUrl('https://browser-http-intake.logs.foo.com/v1/input/xxx')).toBe(true)
       expect(configuration.isIntakeUrl('https://public-trace-http-intake.logs.foo.com/v1/input/xxx')).toBe(true)
 
+      // expect(configuration.isIntakeUrl('https://rum.browser-intake-datadoghq.com/v1/input/xxx')).toBe(true)
+      // expect(configuration.isIntakeUrl('https://logs.browser-intake-datadoghq.com/v1/input/xxx')).toBe(true)
```

Comment:
What is your intent?

Reply:
I forgot to remove these lines, will do now